resources/istio/templates/configmap-cluster.yaml

```diff
@@ -1,8 +1,15 @@
+{{ if not .Values.global.isLocalEnv }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   namespace: {{ .Release.Namespace }}
-  name: kyma-cluster-config
+  name: kyma-istio-control-plane-config
 data:
   config.yaml: |-
 {{.Files.Get "files/kyma_cluster_istio_control_plane.yaml" | printf "%s" | indent 4}}
+{{ if .Values.kyma_istio_control_plane }}
+  #This additional user-provided definition is merged with the default one
+  overrides.yaml: |-
```

Comment:
Note for reviewer: Actual size of the configmap is currently about 40KiB. This is way below the etcd object limit of 1MiB.